### PR TITLE
added background refresh for email and fixed clicking bugs

### LINKED ISSUE #12: Fix defects with login redirect when clicking on "compose"

### PR DESCRIPTION
**Pull request by neelesh17**
Closes #12 #11

### CODE REVIEW

**Comment by neelesh17** [comment]
It's an incomplete PR so do not merge right now.

**Review by saiorama — APPROVED** [review]

**Pull request merged**
